src/main/java/org/moltimate/moltimatebackend/parser/CsaActiveSiteParser.java

```diff
@@ -0,0 +1,81 @@
+package org.moltimate.moltimatebackend.parser;
+
+import com.opencsv.CSVReader;
+import com.opencsv.CSVReaderBuilder;
+import org.moltimate.moltimatebackend.model.ActiveSite;
+import org.moltimate.moltimatebackend.model.Residue;
+import org.springframework.core.io.FileUrlResource;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class CsaActiveSiteParser implements ActiveSiteParser {
+
+    // TODO: don't make web requests for this, use the local file
+    private static final String CSA_CSV_URL = "https://raw.githubusercontent.com/moltimate/moltimate-backend/master/src/main/resources/motifdata/csa_curated_data.csv";
+
+    /**
+     * Retrieve all active sites from the Catalytic Site Atlas.
+     *
+     * @return A list of ActiveSites
+     */
+    public List<ActiveSite> parseMotifs() {
+        try {
+            Reader reader = new InputStreamReader(new FileUrlResource(new URL(CSA_CSV_URL)).getInputStream());
+            CSVReader csvReader = new CSVReaderBuilder(reader).withSkipLines(1)
+                    .build();
+
+            List<ActiveSite> activeSites = new ArrayList<>();
+            ActiveSite nextSite;
+            while ((nextSite = readNextCsaActiveSite(csvReader)) != null) {
+                activeSites.add(nextSite);
+            }
+
+            return activeSites;
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
+        return Collections.emptyList();
```

Comment:
Can this be put inside of the catch since it should only occur when an error occurs